src/output.rs

```diff
@@ -164,6 +164,8 @@ pub(crate) struct Inner {
     pub(crate) description: String,
     #[cfg(feature = "wayland_frontend")]
     pub(crate) instances: Vec<WlOutput>,
+    #[cfg(feature = "wayland_frontend")]
+    pub(crate) handle: Option<WeakHandle>,
```

Comment:
Could we get a comment describing that this is set and used by wayland::output, perhaps even group it together with `instances` and  `xdg_output` on the bottom of the struct definition, so the comment can apply to all of them?

Asking mostly because It took me a sec to guess that this is most likely set to Some by Wayland module, so someone not as familiar with the codebase might be quite confused by this.